### ISSUE TITLE
D3.11.4.2: Investigate and implement dynamic source cross-match service

### ISSUE THREAD

**Issue by Onoddil** [lsst-uk/Cross-Match-WP3.11#10]
Another placeholder goal at present, in parallel to #9 is the issue of whether users will be able to upload small catalogues of objects and obtain dynamic cross-match results on-the-fly. This will require investigation and is currently a stretch goal before its confirmation and any APIs are devised.

**Comment by tms-epcc** [comment]
03/NOV/22
- would like to talk to Lasair team about this.
- Mario Juric (LINCC) would like to discuss this further as well. Also Leanne Guy would like this for Rubin-Euclid as well as Isobel Hook. 
- other inkinds may also be interested https://www.lsst.org/in-kind-program/programs

**Comment by tms-epcc** [comment]
05/APR/23
- unclear whether this will be tackled in the 6 months starting April'23

**Comment by tms-epcc** [comment]
23/MAY/24
- also agreed to move backlog
